default.nix

```diff
@@ -10,13 +10,15 @@ let
     rev = "94f779a7";
     sha256 = "16lxilng5q8fr2ll6h4hf7wlvac6nmw4cx10cbgzj5ks090bl97r";
     patchFile = ./nix/fix-mainnet.patch;
-    protoName = "proto_005_PsBabyM1";
+    protoName = "005_PsBabyM1";
+    binarySuffix = "005-PsBabyM1";
   };
   babylonnet = {
     rev = "b8731913";
     sha256 = "1pakf1s6bg76fq42mb8fj1immz9g9wwimd522cpx8k28zf0hkl5i";
     patchFile = ./nix/fix-babylonnet.patch;
-    protoName = "proto_005_PsBabyM1";
+    protoName = "005_PsBabyM1";
+    binarySuffix = "005-PsBabyM1";
```

Comment:
It seems to be the same as `protoName` except that `_` is replaced with `-`. I don't know what these values mean, but I wonder if it is accidentally so or `binarySuffix` is always equal to `protoName` with `_` replaced with `-`. In the latter case it seems better to define `binarySuffix` in terms of `protoName` (so that we'll have to update less things in case of new protocol), but there is another thing I don't know: whether it is hard to do that in `nix`. So that's up to you.

Reply:
Fair enough